toolflow/vivado/common/ip.tcl

```diff
@@ -540,6 +540,15 @@ namespace eval ::tapasco::ip {
             set kid [dict get [::tapasco::get_composition] $kind id]
             set vlnv [dict get [::tapasco::get_composition] $kind vlnv]
 
+            set ipecfeature [tapasco::get_feature "IPEC"]
```

Comment:
Is there a reason why this is separated into a plugin?

Reply:
An ipec pe has multiple controlregister interfaces, each associated with an internal PE.
This replaces the vlnv and id of the ipec pe with the vlnv and id of one of the PEs within the ipec pe.

The dict containing the replacement information must come from somewhere, I assumed a plugin is the right choice.

Reply:
Sorry, I meant to ask why this is **not** separated into a plugin.
I think there should not be any feature-specific code in the "core" and it would be better to add a plugin hook there.

Reply:
I moved that part into a plugin